src/vaults/Vault.sol

```diff
@@ -208,10 +208,9 @@ abstract contract Vault is IVault, CoreReference, ReentrancyGuardUpgradeable, Va
         }
 
         // Add it to their withdraw request and log the epoch
-        Request storage userWithdrawRequest = assetData.withdrawRequests[msg.sender];
-        userWithdrawRequest.amount = _withdrawAmountDay0 + req.amount;
+        req.amount = _withdrawAmountDay0 + req.amount;
```

Comment:
tested it locally, this is the most gas efficient arrangement of variables

Reply:
nice - also much more readable this way
`memory req` threw me off when I was looking at this